src/PhpSpreadsheet/Settings.php

```diff
@@ -166,23 +161,19 @@ public static function getCache()
         return self::$cache;
     }
 
-    /**
-     * Set the HTTP client implementation to be used for network request.
-     */
-    public static function setHttpClient(ClientInterface $httpClient): void
```

Comment:
This should probably be kept as:

```php
public static function setHttpClient(ClientInterface $httpClient, RequestFactoryInterface $requestFactory): void
```

Because both parameters must be set, in a coherent way, in order for WEBSERVICE to work.